subsys/net/openthread/Kconfig.defconfig

```diff
@@ -10,6 +10,18 @@
 
 if NET_L2_OPENTHREAD
 
+if OPENTHREAD_THREAD_VERSION_1_3
```

Comment:
What about moving this block after `OPENTHREAD_STACK_VERSION` one?

Reply:
We can also restrict here the usage of libraries when coprocessor is being enabled.